.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- Kindly, Create A `test.md` file with the text "This file should be ignored by git" and ignore it by including it in the `.gitignore` file to fit the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) 👍